### PR TITLE
Update project docs

### PR DESCRIPTION
**Pull request by brynrhodes**
Update project and process documentation for the repository to provide some formal description of the informal processes we've been following so far.

### CODE REVIEW

File: CHANGE_MANAGEMENT.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+## Roles
+### Stakeholders
+Stakeholders are community members who have a need for the tooling provided by this project. They are the most important members of the community and without them the project would have no purpose. Anyone can be a stakeholder; there are no special requirements.
+
+Stakeholder participation is on a voluntary basis, but the project asks its stakeholders to participate in the project and community as much as possible. Stakeholder contributions enable the project team to ensure that they are satisfying the requirements of those stakeholders. Common stakeholder contributions include (but are not limited to):
+* Evangelism of the project (e.g. a link on a website and word-of-mouth awareness)
+* Informing developers of strengths and weaknesses from a new stakeholder perspective
+* Providing moral support (a ‘thank you’ goes a long way)
+* Providing financial support (the project is open source, but capacity is a constant struggle)
```
Comment (cmoesel) [inline CHANGE_MANAGEMENT.md]: How does this happen?  If there is not a clear process for providing financial support, then this should probably be removed.

File: CHANGE_MANAGEMENT.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+* Reporting bugs
+* Identifying requirements
+* Providing graphics and web design
+* Programming
+* Assisting with project infrastructure
+* Writing documentation
+* Fixing bugs
+* Adding features
+
+Contributors engage with the project through the issue tracker and mailing list, or by writing or editing documentation. They submit changes to the project itself via pull requests, which will be considered for inclusion in the project by existing committers (see next section). The mailing list is the most appropriate place to ask for help when making that first contribution.
```
Comment (cmoesel) [inline CHANGE_MANAGEMENT.md]: We don't have a mailing list, do we?  If so, I'm not on it!

File: CHANGE_MANAGEMENT.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+As contributors gain experience and familiarity with the project, their profile within, and commitment to, the community will increase. At some stage, they may find themselves being nominated for committership.
+
+### Committers
+Committers are community members who have shown that they are committed to the continued development of the project through ongoing engagement with the community. New committers can be nominated by any existing committer. Once they have been nominated, there will be a vote by the project management committee (PMC; see below). Committer voting is one of the few activities that takes place on the project’s private management list. This is to allow PMC members to freely express their opinions about a nominee without causing embarrassment. Once the vote has been held, the aggregated voting results are published on the public mailing list. The nominee is entitled to request an explanation of any ‘no’ votes against them, regardless of the outcome of the vote. This explanation will be provided by the PMC Chair (see below) and will be anonymous and constructive in nature.
```
Comment (cmoesel) [inline CHANGE_MANAGEMENT.md]: Again, we don't really have a _private management (email) list_ or a _public mailing list_.

File: CHANGE_MANAGEMENT.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+* Discussion
+* Vote (if consensus is not reached through discussion)
+* Decision
+
+Any community member can make a proposal for consideration by the community. In order to initiate a discussion about a new idea, they should submit an issue to the project issue tracker, or a pull request implementing the idea. This will prompt a review and, if necessary, a discussion of the idea. The goal of this review and discussion is to gain approval for the contribution.
+
+In general, as long as nobody explicitly opposes a proposal or change, and the functionality is aligned with the CQL specification, it is recognized as having the support of the community. This is called lazy consensus - that is, those who have not stated their opinion explicitly have implicitly agreed to the implementation of the proposal.
+
+For lazy consensus to be effective, it is necessary to allow at least 1 week before assuming that there are no objections to the proposal. This requirement ensures that everyone is given enough time to read, digest and respond to the proposal. This time period is chosen so as to be as inclusive as possible of all participants, regardless of their location and time commitments.
```
Comment (cmoesel) [inline CHANGE_MANAGEMENT.md]: It's unclear to me -- does this 1 week wait only apply to significant changes?  Would a feature that implements a defined part of the spec or fixes a known bug we subject to this 1-week waiting period?  Or is this only for more significant items?

File: Src/java/README.md [hunk 5]
```diff
@@ -1,12 +1,17 @@
...
+This is a multi-module project for CQL java libraries and tooling applications.
 It contains the following sub-projects:
 
 * **cql:** generates and builds Java lexers, parsers, listeners, and visitors using the CQL ANTLR4 grammar
-* **quick:** generates and builds Java classes based on the QUICK XML schema
+* **model:** generates and builds Java classes based on the ELM Model Info schema and CQL base type system
 * **elm:** generates and builds Java classes based on the ELM XML schema
-* **cql-to-elm:** generates Expression Logical Model (ELM) XML from CQL source
+* **qdm:** contains schema and model info resources for QDM (4.2, 5.0, 5.0.1, 5.0.2, 5.3)
+* **quick:** contains schema and model info resources for QUICK and FHIR, DSTU2 (1.0.2), and STU3 (1.4, 1.6, 1.8, and 3.0.1)
```
Comment (cmoesel) [inline Src/java/README.md]: I think we eventually need to give fhir its own space.  (Just a comment; nothing to be acted on right now).

File: CHANGE_MANAGEMENT.md (hunk 1 above)
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md]: Agreed, will remove.

File: CHANGE_MANAGEMENT.md (hunk 2 above)
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md]: Not yet, just created. I'll add the link to the document.

File: CHANGE_MANAGEMENT.md (hunk 3 above)
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md]: Just created a committers private mail, I'll invite you.

File: CHANGE_MANAGEMENT.md (hunk 4 above)
Comment (brynrhodes) [inline CHANGE_MANAGEMENT.md]: Agreed, and I wondered about this as I put it in (it was 3 days in the source document). I think we need to be a bit more flexible here, that's why I changed to 1 week, but maybe a better approach would be something like "a reasonable amount of time based on the significance of the issue."

File: Src/java/README.md (hunk 5 above)
Comment (brynrhodes) [inline Src/java/README.md]: Agreed, I've thought that many times.

**Pull request merged**